db/migrations/00002_feedback_tables.go

```diff
@@ -0,0 +1,101 @@
+package migrations
+
+import (
+	"database/sql"
+	"github.com/pressly/goose"
+	"github.com/jinzhu/gorm"
+
+	"github.com/iReflect/reflect-app/apps/feedback/models"
```

Comment:
Migrations should use frozen models and not directly point to current models file, as that may break this migration/future migrations.

Reply:
done